evap/evaluation/tests/test_misc.py

```diff
@@ -14,7 +14,7 @@
 
 
 @override_settings(INSTITUTION_EMAIL_DOMAINS=["institution.com", "student.institution.com"])
-class SampleXlsTests(WebTestStaffMode):
+class SampleXlsxTests(WebTestStaffMode):
```

Comment:
For the class and test names, instead of changing them from "xls" to "xlsx", I'd say we should change them to something independent such as "file".